tests/nodes/NodeConnectionManager.lifecycle.test.ts

```diff
@@ -274,22 +310,19 @@ describe(`${NodeConnectionManager.name} lifecycle test`, () => {
       await taskManager.startProcessing();
       // @ts-ignore: kidnap connections
       const connections = nodeConnectionManager.connections;
-      // @ts-ignore: kidnap connectionLocks
-      const connectionLocks = nodeConnectionManager.connectionLocks;
       const initialConnection = connections.get(remoteNodeIdString1);
       expect(initialConnection).toBeUndefined();
       await nodeConnectionManager.withConnF(remoteNodeId1, async () => {
-        expect(connectionLocks.isLocked(remoteNodeIdString1)).toBe(true);
+        // Do nothing
```

Comment:
Why run this if it does nothing?

Reply:
It creates the connection for us. the expectation of locking wasn't relevant anymore.

Reply:
I see, so there's no more direct way of connecting to the other side?

Reply:
Does the test require this connection?

Reply:
Yes, we're testing if a connection is created here. Currently `withConnF` is the only way of establishing a connection.

Reply:
Ok sure. But it is also possible to directly use the resource allocation API too right? The `with*` variants are usually convenience wrappers.